src/pages/AddTenant.tsx

```diff
@@ -0,0 +1,206 @@
+import { useEffect, useState } from "react";
+import DropDown from "../components/DropDown";
+import { DropDownOption, DropDownVarient } from "../types/DropDown";
+import NewTenantHeader from "../components/NewTenantHeader";
+import DatePicker from "../components/DatePicker";
+import FileUploader from "../components/FileUploader";
+import AddTable from "../components/AddTable";
+
+interface TowerData {
+  name: string;
+  title: string;
+  dropDownOptions: DropDownOption[];
+}
+
+interface ElevatorData {
+  name: string;
+  title: string;
+  dropDownOptions: DropDownOption[];
+}
+
+const AddTenant = () => {
+  const [towerData, setTowerData] = useState<TowerData | null>(null);
+  const [elevatorData, setElevatorData] = useState<ElevatorData | null>(null);
+  const [selectedData, setSelectedDate] = useState<string | null>(null);
+  const [selectedElevator, setSelectedElevator] = useState<string | null>(null);
+
+  useEffect(() => {
+    fetch("/assets/towers.json")
+      .then((response) => response.json())
+      .then((json) => {
+        const dataOfTower = json.find(
+          (item: TowerData) => item.name === "tower"
+        );
+        setTowerData(dataOfTower || null);
+      })
+      .catch((error) => console.error("Error in fetching data", error));
+  });
+
+  useEffect(() => {
+    fetch("/assets/elevator.json")
+      .then((response) => response.json())
+      .then((json) => {
+        const dataOfElevator = json.find(
+          (item: ElevatorData) => item.name === "elevator"
+        );
+        setElevatorData(dataOfElevator || null);
+      })
+      .catch((error) => console.error("Error in fetching data", error));
+  }, []);
+
+  const isElevatorSlotVisible = selectedData && selectedElevator;
+
+  return (
+    <div className="bg-gray-100 h-full">
+      <NewTenantHeader />
+      <div className="py-8 px-56">
+        <div>
+          <h5 className="font-semibold text-xl leading-7">Tenant Details</h5>
+          <div className="mt-5">
+            <div className="flex flex-col gap-2 w-full">
+              <label className="font-medium text-sm leading-5 text-gray-900">
+                Apartment
+              </label>
+              <div className="flex w-full">
+                <div className="flex">
+                  <DropDown
+                    hasTitle
+                    onChange={() => {}}
+                    data={
+                      towerData || { title: "Loading..", dropDownOptions: [] }
+                    }
+                    styles={{
+                      borderTopRightRadius: "0px",
+                      borderBottomRightRadius: "0px",
+                    }}
+                  />
+                </div>
+                <div className="flex">
+                  <DropDown
+                    hasTitle
+                    onChange={() => {}}
+                    data={
+                      towerData || { title: "Loading..", dropDownOptions: [] }
```

Comment:
Actually, it's apartment data, not tower data. The apartment dropdown should list apartments based on the selected tower.